src/components/LandingPage/TwitterWidget.tsx

```diff
@@ -33,38 +38,55 @@ const TwitterWidget: React.FC = () => {
   }
 
   return (
-    <div className="flex items-center justify-around h-auto p-36">
-      <button
-        type="button"
-        className="m-3 rounded-full h-12 w-12 border shadow-lg"
-        onClick={previousTweet}
-      >
-        {'<'}
-      </button>
-      {arr.map(
-        (tweetId, index) =>
-          index === firstTweetId && (
-            <div className="flex justify-center space-x-10 w-3/4">
-              <TwitterTweetEmbed
-                tweetId={arr[firstTweetId]}
-                placeholder="Loading Tweet/"
-                options={{ width: 550, cards: 'hidden' }}
-              />
-              <TwitterTweetEmbed
-                tweetId={arr[secondTweetId]}
-                placeholder="Loading Tweet"
-                options={{ width: 400, cards: 'hidden' }}
-              />
-            </div>
-          )
-      )}
-      <button
-        type="button"
-        className="m-3 rounded-full h-12 w-12 border shadow-lg"
-        onClick={nextTweet}
-      >
-        {'>'}
-      </button>
+    <div>
+      <div className="flex items-center justify-around h-auto pt-36 pb-8">
+        <button
+          type="button"
+          className="m-3 rounded-full h-12 w-12 border shadow-lg"
+          onClick={previousTweet}
+        >
+          {'<'}
+        </button>
+        {arr.map(
```

Comment:
@alaa-yahia I didn't notice .. but update the name of the variable to something related `const tweets = [` instead of `arr`